<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DRuGLORD" failures="9" tests="16" name="Default test" time="61.873" errors="0" timestamp="2025-08-12T13:01:15 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.071"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.196"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.815">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.227">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.462">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.218">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.319">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.287">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.285">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.167">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.196">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.159"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.407"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.085"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.843"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.444"/>
</testsuite> <!-- Default test -->
